core/src/test/scala/com/evolutiongaming/catshelper/SerialKeyTest.scala

```diff
@@ -378,7 +378,7 @@ object SerialKeyTest {
 
   object Queue {
 
-    def of[F[_]: Concurrent: Parallel, K: Hash, A]: F[Queue[F, K, A]] = {
+    def of[F[_]: Concurrent, K: Hash, A]: F[Queue[F, K, A]] = {
```

Comment:
these are breaking changes and not relevant to original problem, could you please deprecate this method and create a new one as separate MR? as I can accept other part already :)